### PR TITLE
Drizzle 

### PR DESCRIPTION
**Pull request by itsTony4dev**

### CODE REVIEW

File: packages/backend/src/drizzle/schema.ts [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+    speakerName: varchar('speaker_name', { length: 255 }),
+    speakerDescription: varchar('speaker_description', { length: 255 }),
+    speakerProfileUrl: varchar('speaker_profile_url', { length: 255 }),
+    typeId: integer('type_id').references(() => jobType.id),
+    tags: text('tags'),
+    createdAt: timestamp('created_at'),
+    updatedAt: timestamp('updated_at'),
+  },
+  (table) => ({
+    eventDateIdx: index('event_date_idx').on(table.date),
```
Comment (Hkataya) [inline packages/backend/src/drizzle/schema.ts:157]: Why are we indexing event date

File: packages/backend/src/drizzle/schema.ts [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+    eventTypeIdx: index('event_type_idx').on(table.typeId),
```
Comment (Hkataya) [inline packages/backend/src/drizzle/schema.ts:158]: Same here, i would add a fulltext index to event title maybe but not the rest

**Review by Hkataya — CHANGES_REQUESTED** [review]

**Comment by Hkataya** [comment]
general comments:

- Move all the files into a folder named db. 
- The file named db rename to index.ts

File: packages/backend/src/drizzle/migrations/meta/_journal.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
```
[the remark this answers is not in the document]
Comment (itsTony4dev) [reply packages/backend/src/drizzle/migrations/meta/_journal.json:1]: typically keeps track of which migrations have been applied to the database. it's crucial for maintaining the state of database schema.
Without this file you might risk data integrity issues if migrations are run out of order or duplicated.

File: packages/backend/drizzle.config.ts
Comment (Hkataya) [inline packages/backend/drizzle.config.ts:1]: @itsTony4dev Seems like we didnt use this config anywhere ?

File: packages/backend/drizzle.config.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import * as dotenv from 'dotenv';
```
Comment (Hkataya) [inline packages/backend/drizzle.config.ts:1]: Also Rename drizzle folder to db. And db file to index

**Review by Hkataya — CHANGES_REQUESTED** [review]

**Review by Hkataya — APPROVED** [review]

**Pull request merged**